### PR TITLE
Add `UKCC1` tournament article and other `UKCC` prerequisites

### PR DESCRIPTION
**Pull request by Bazzadwarf**
Since UKCC 10 is starting up soon my plans to get UKCC added has been bumped up a bit.

- [x] host fact check

### CODE REVIEW

File: wiki/Tournaments/UKCC/1/en.md [hunk 1]
```diff
@@ -0,0 +1,651 @@
...
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2016-04-25/2016-06-12 |
+| Live drawings | 2016-06-13 |
+| Qualifiers | 2016-06-18/2016-06-19 |
+| Round of 64 | 2016-06-25/2016-06-26 |
+| Group Stage | 2016-07-02/2016-07-03 |
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/1/en.md:20]:
```suggestion
| Group stage | 2016-07-02/2016-07-03 |
```

File: wiki/Tournaments/UKCC/1/en.md [hunk 2]
```diff
@@ -0,0 +1,651 @@
...
+- DoubleTime
+  - [BRIGHT - 1 year 2 months 20 days (Athena Tennos) [Athena]](https://osu.ppy.sh/beatmapsets/57546#osu/204428)
+  - [Lily - Marie-Luise (spboxer3) [Insane]](https://osu.ppy.sh/beatmapsets/31408#osu/103221)
+  - [SMiLE.dk - Golden Sky (meiikyuu) [Insane]](https://osu.ppy.sh/beatmapsets/50587#osu/155404)
+- FreeMod
+  - [Cutline - Die For You (Shock One Remix) (Katty Pie) [Insane]](https://osu.ppy.sh/beatmapsets/155285#osu/380964)
+  - [ALiCE'S EMOTiON - LUNA (Frostmourne) [Lunatic]](https://osu.ppy.sh/beatmapsets/47124#osu/152180)
+  - [IOSYS - Chanteikku Sanyousei no Itazura Daisensou (Kochiya Sanae) [Crazy Jay]](https://osu.ppy.sh/beatmapsets/24448#osu/91462)
+- Tiebreaker
+  - **[Shimotsuki Haruka - Youka to Yumeutsutsu (Naitoshi) (Awakening)](https://osu.ppy.sh/beatmapsets/324587#osu/721158)**
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/1/en.md:248]:
```suggestion
  - **[Shimotsuki Haruka - Youka to Yumeutsutsu (Naitoshi) [Awakening]](https://osu.ppy.sh/beatmapsets/324587#osu/721158)**
```

File: wiki/Tournaments/UKCC/1/en.md [hunk 3]
```diff
@@ -0,0 +1,651 @@
...
+- DoubleTime
+  - [Ellie Goulding - Love Me Like You Do (Gero) [Soulmates]](https://osu.ppy.sh/beatmapsets/280639#osu/634965)
+  - [Liviu Hodor feat. Mona - Sweet Love (kazumikos) [Insane]](https://osu.ppy.sh/beatmapsets/55117#osu/167378)
+- FreeMod
+  - [Kozato - 45nen no Yukizakura (LoliSora) [Another]](https://osu.ppy.sh/beatmapsets/50771#osu/155929)
+  - [Glamour of the Kill - Feeling Alive (Katty Pie) [Insane]](https://osu.ppy.sh/beatmapsets/73333#osu/208927)
+- Tiebreaker
+  - **[Receptor - Redstar (fergas) [Glory]](https://osu.ppy.sh/beatmapsets/184301#osu/502414)**
+
+## Match Results
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/1/en.md:294]:
```suggestion
## Match results
```

File: wiki/Tournaments/UKCC/1/en.md [hunk 4]
```diff
@@ -0,0 +1,651 @@
...
+| **helix** ![][flag_GB] | **5** | 0 | ![][flag_GB] Woffel |
+
+#### Losers Bracket Round 4
+
+Wednesdady, 27 July 2016:
+
+| Player 1 |  |  | Player 2 | Match info |
+| --: | :-: | :-: | :-- | :-- |
+| Mafham ![][flag_GB] | 0 | **5** | ![][flag_GB] **OPJames** |  |
+| helix ![][flag_GB] | 0 | **0** | ![][flag_GB] **Bubbleman** | *no score recorded* |
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/1/en.md:368]:
```suggestion
| helix ![][flag_GB] | -1 | **0** | ![][flag_GB] **Bubbleman** | *no score recorded* |
```

File: wiki/Tournaments/UKCC/1/en.md [hunk 5]
```diff
@@ -0,0 +1,651 @@
...
+| **Xim** ![][flag_GB] | **4** | 0 | ![][flag_GB] Woffel |
+| **Bubbleman** ![][flag_GB] | **4** | 0 | ![][flag_GB] BAEx |
+
+Friday, 15 July 2016:
+
+| Player 1 |  |  | Player 2 |
+| --: | :-: | :-: | :-- |
+| YunoWhoItIs ![][flag_GB] | 0 | **4** | ![][flag_GB] **helix** |
+
+### Group Stage
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/1/en.md:455]:
```suggestion
### Group stage
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/UKCC/1/en.md [hunk 6]
```diff
@@ -0,0 +1,651 @@
...
+#### Group A
+
+| Player 1 |  |  | Player 2 | Match info |
+| --: | :-: | :-: | :-- | :-- |
+| **Doomsday** ![][flag_GB] | **4** | 1 | ![][flag_GB] Taiitoo |  |
+| **Doomsday** ![][flag_GB] | **4** | 1 | ![][flag_GB] helix |  |
+| Pyrooo ![][flag_GB] | 2 | **4** | ![][flag_GB] **Taiitoo** |  |
+| **Doomsday** ![][flag_GB] | **4** | 0 | ![][flag_GB] Pyrooo |  |
+| **helix** ![][flag_GB] | **4** | 3 | ![][flag_GB] Taiitoo |  |
+| **helix** ![][flag_GB] | **0** | 0 | ![][flag_GB] Pyrooo | *no score recorded* |
```
Comment (Bazzadwarf) [inline wiki/Tournaments/UKCC/1/en.md:466]:
```suggestion
| **helix** ![][flag_GB] | **0** | -1 | ![][flag_GB] Pyrooo | *no score recorded* |
```

File: wiki/Tournaments/en.md [hunk 7]
```diff
@@ -311,6 +311,22 @@ Unofficial tournaments/competitions hosted by the communities.
...
+| [UK Community Cup 6](UKCC/6) | 2019-02-23 | 2019-04-07 | Question Mark | 10man? | Boom! Tetris for Benbow! |
+| [UK Community Cup 7](UKCC/7) | 2019-09-07 | 2019-10-27 | ![][flag_GB] [Bubbleman](https://osu.ppy.sh/users/5182050) | ![][flag_GB] [Spare](https://osu.ppy.sh/users/2204373) | ![][flag_GB] [AJT](https://osu.ppy.sh/users/3181083) |
+| [UK Community Cup 8](UKCC/8) | 2020-03-28 | 2020-05-17 | 10man? | will is minecart | Gaurdians of Valhalla |
+| [UK Community Cup 9](UKCC/9) | 2020-08-29 | 2020-10-11 | ![][flag_GB] [Bubbleman](https://osu.ppy.sh/users/5182050) | ![][flag_GB] [majoreh](https://osu.ppy.sh/users/7959222) | ![][flag_GB] [Kingling](https://osu.ppy.sh/users/7010761) |
+| [UK Community Cup X](UKCC/X) | 2021-02-06 | 2021-04-12 | ![][flag_GB] [Kingling](https://osu.ppy.sh/users/7010761) | ![][flag_GB] [Voss](https://osu.ppy.sh/users/7657761) | ![][flag_GB] [Kerantina](https://osu.ppy.sh/users/7516484) |
+| [UK Community Cup 10](UKCC/10) | 2021-09-11 | 2021-10-31 | *tdb* | *tdb* | *tdb* |
```
[the remark this answers is not in the document]
Comment (Bazzadwarf) [reply wiki/Tournaments/en.md]: holy crap I am blind

**Review by TPGPL — APPROVED** [review]

**Pull request merged**